__tests__/app.test.js

```diff
@@ -131,6 +131,58 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with array of comments for the article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.comments)).toBe(true);
+        expect(body.comments.length).toBeGreaterThan(0);
+        expect(body.comments[0]).toHaveProperty("comment_id");
+        expect(body.comments[0]).toHaveProperty("votes");
+        expect(body.comments[0]).toHaveProperty("created_at");
+        expect(body.comments[0]).toHaveProperty("author");
+        expect(body.comments[0]).toHaveProperty("body");
+        expect(body.comments[0]).toHaveProperty("article_id");
```

Comment:
It's worth making an assertion on the values as well as the properties. At least check that each comment that is returned is for the correct article